crates/copilot/src/copilot_chat.rs

```diff
@@ -123,10 +135,12 @@ impl Model {
         match self {
             Self::Gpt4o => 64_000,
             Self::Gpt4 => 32_768,
-            Self::Gpt4_1 => 1_047_576,
+            Self::Gpt4_1 => 128_000,
```

Comment:
Was this intentional?

Reply:
https://github.com/zed-industries/zed/pull/28881#discussion_r2048497154

Reply:
Thanks, and sorry for not reading the conversation. Just looked at the diff.